pages/1.13/security/oss/managing-users/external-users/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle: External users
+title: External users
+excerpt: Managing external DC/OS users
+menuWeight: 10
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+An external user is automatically created for the first user who logs in to the DC/OS cluster via a single sign-on flow through Auth0.
+
+# Log in external users via the web interface
+
+1.  Launch the DC/OS web interface.
+
+2.  Log in through a single sign-on flow using the identity provider of your choice (Google, GitHub, and Microsoft).
+
+**NOTE**: Additional users must be added by another authorized user before they can log in to DC/OS.
+
+
+# Log in external users via the DC/OS CLI
+
+1.  To log in to the DC/OS CLI, enter the [auth login](/1.13/cli/command-reference/dcos-auth/dcos-auth-login/) command:
+
+```bash
+dcos auth login --provider dcos-oidc-auth0
+If your browser didn't open, please go to the following link:
+
+    http://172.17.0.2/login?redirect_uri=urn:ietf:wg:oauth:2.0:oob
+
+Enter OpenID Connect ID Token: 
+>
+```
+
+2. Follow the displayed instructions to trigger the browser login flow.
+
+3. Copy the `OpenID Connect ID token` from the modal dialog that appears in the browser.
```

Comment:
"After logging in through your provider, copy...".

Also, "modal dialog" sounds a bit technical. Maybe just "dialog", or "copy the `OpenID Connect ID token` that appears in the browser."

Also, technically, I think we decide that Auth0 is not OpenID Connect, so we should probably just call it an Identity Token. (But, more importantly, it should be consistent with what the dialog calls it).

Reply:
That's a funny one because the CLI calls it `OpenID Connect ID Token` :)

Reply:
> I think we decide that Auth0 is not OpenID Connect, so we should probably just call it an Identity Token

When we look at the complete flow it's none of authorization code flow, implicit flow, but the token emitted by Auth0 still appears to be a standard-compliant OpenID Connect ID Token :-)